### DISCUSSION TITLE
How Mod Package work ??

### DISCUSSION THREAD

**Discussion by michalss** [nesrak1/UABEA#373] [General]
Hi,

Can anyone please explain how Mod Package works in general ? Its very simple to make emip file. But how to make the game to read it ?

Regards

**Comment by nesrak1** [comment]
I wouldn't suggest using it. It was unfinished and I've explained the problems with it in other github issues in this repo.

**Reply by michalss** [reply]
ok thx i wont use this then. But can i use cli commands instead then pls ? I know there is some support but it never worked. I was trying to import data to resourse.asset file without the success.
